akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -56,14 +56,13 @@ sealed trait Multipart extends jm.Multipart {
   /**
    * Creates a [[akka.http.scaladsl.model.MessageEntity]] from this multipart object.
    */
-  def toEntity(
-    charset:  HttpCharset = HttpCharsets.`UTF-8`,
-    boundary: String      = BodyPartRenderer.randomBoundary())(implicit log: LoggingAdapter = NoLogging): MessageEntity = {
+  def toEntityWithLog(
```

Comment:
Not sure how you'd like to deal with my introduced `toEntityWithLog` naming issue, so I just left it as 'awful' for the moment